_docs/client/themes.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: documentation
+title: Themes
+description: Add your description here
```

Comment:
I see now that all pages have this... Urgh. I'd say we should start changing that but in a later comment I mention that I'd rather see this in the `theme` section of the configuration doc.